vernac/declareUniv.ml

```diff
@@ -10,6 +10,17 @@
 
 open Names
 
+(* object_kind , id *)
```

Comment:
This comment should rather be in the mli

Reply:
Done, thanks [I'd like to get rid of this anyways]